source/ScrollSync/ScrollSync.js

```diff
@@ -67,5 +101,18 @@ export default class ScrollSync extends React.PureComponent {
       scrollTop,
       scrollWidth,
     });
+    this._elements.forEach(element => {
+      if (element != null && element.scrollToPosition)
+        element.scrollToPosition({scrollLeft, scrollTop});
+    });
+  }
+
+  _registerChild(element: ScrollableElement | null) {
+    if (element == null || !element.scrollToPosition) {
+      console.warn(
+        "ScrollSync registerChild expects to be passed a component with 'scrollToPosition' function",
+      );
+    }
+    this._elements.push(element);
```

Comment:
Should we register if its falsey?

Reply:
You're right, probably not, there is no much sense in keeping the element in that case. I'll fix this!